### DISCUSSION TITLE
It would be more perfect to replace this icon with the icon of the website.

### DISCUSSION THREAD

**Discussion by zhouxinghong** [YU000jp/logseq-plugin-confirmation-hyperlink#17] [Announcements]
![image](https://github.com/YU000jp/logseq-plugin-confirmation-hyperlink/assets/54253524/e4ce95e1-8711-4f25-953a-0913c8ae8493)
It would be more perfect to replace this icon with the icon of the website.
The specific instructions are as follows:
Allows the user to turn display of the favicon on or off.

**Comment by YU000jp** [comment]
Please use *Awesome Link* plugin, as it contains that feature.
https://github.com/yoyurec/logseq-awesome-links
